internal/controller/ovnk_ipfix_collector.go

```diff
@@ -0,0 +1,291 @@
+// Copyright 2025 Illumio, Inc. All Rights Reserved.
+
+package controller
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"errors"
+	"net"
+	"os"
+	"time"
+
+	pb "github.com/illumio/cloud-operator/api/illumio/cloud/k8sclustersync/v1"
+	netflows "github.com/netsampler/goflow2/decoders/netflow"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OVNKFlow represents a flow captured from OVN-Kubernetes.
+type OVNKFlow struct {
+	SourceIP        string
+	DestinationIP   string
+	SourcePort      uint16
+	DestinationPort uint16
+	Protocol        string
+	IPVersion       string
+	StartTimestamp  *timestamppb.Timestamp
```

Comment:
Also capture the end timestamp. It's not a significant effort to do that.
We're not sending the end timestamp to CloudSecure yet, but it would make sense to do that since we're aggregating flows in cloud-operator now.